koku/masu/processor/aws/aws_report_parquet_summary_updater.py

```diff
@@ -102,6 +105,24 @@ def update_summary_tables(self, start_date, end_date, **kwargs):
             accessor.populate_tags_summary_table(bill_ids, start_date, end_date)
             accessor.populate_category_summary_table(bill_ids, start_date, end_date)
             accessor.update_line_item_daily_summary_with_tag_mapping(start_date, end_date, bill_ids)
+
+            # Populate ec2 compute summary table if feature is enabled for schema
+            if is_feature_cost_4403_ec2_compute_cost_enabled(self._schema):
+                LOG.info(f"AWS EC2 compute summary is enabled for schema: {self._schema}")
+
+                # Ensure start_date is first day of the month
+                month_start_date = start_date.replace(day=1)
+
+                # Delete records from the EC2 compute summary table for a specified source and date range before insert
+                accessor.delete_line_item_daily_summary_entries_for_date_range_raw(
+                    self._provider.uuid, month_start_date, end_date, table=AWS_CUR_TABLE_MAP["ec2_compute_summary"]
+                )
```

Comment:
Do we need the source-uuid filter?
```suggestion
                accessor.delete_line_item_daily_summary_entries_for_date_range_raw(
                    self._provider.uuid, month_start_date, end_date, table=AWS_CUR_TABLE_MAP["ec2_compute_summary"], filters={"source_uuid": self._provider.uuid}
                )
```

Reply:
💯 yes we do, thanks for catching this.